torch/quantization/quantize_fx.py

```diff
@@ -12,6 +12,16 @@ def _check_is_graph_module(model):
             'Got type:' + str(type(model)) + ' Please make ' +
             'sure to follow the tutorials.')
 
+def _swap_ff_with_fxff(model):
```

Comment:
would it make sense to also delete the old modules here?

Reply:
they are removed from model._modules when we reassign, is that necessary?

Reply:
optional IMO, I don't feel strongly.  It would be defensive if something outside of the model depends on the object.

Reply:
sure, done.